packages/basic/src/typography.scss

```
@@ -0,0 +1,383 @@
+    --prosekit-border-color: color-mix(in srgb, CanvasText 15%, Canvas);
+    --prosekit-blockquote-color: color-mix(in srgb, CanvasText 40%, Canvas);
+    --prosekit-autocomplete-color: color-mix(in srgb, CanvasText 5%, Canvas);
+  }
+  // #endregion Variables
+
+  // #region Basic
+  & {
+    // Disable margin collapsing
+    display: flex;
+    flex-direction: column;
+  }
```

Stylelint (17.4.0)

[error] Unexpected redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)